v 2.3.3 –  Cannot find module 'react-native-capture-protection/jest/capture-protection-mock'

Hello

After upgrading to version 2.3.3 jest is erroring saying it cannot find the mock module

```
Cannot find module 'react-native-capture-protection/jest/capture-protection-mock'
```

Maybe it is related to [this change](https://github.com/wn-na/react-native-capture-protection/compare/v2.3.2...v2.3.3) in `package.json`?

<img width="2508" height="1034" alt="Image" src="https://github.com/user-attachments/assets/bea0d6f8-48dc-4243-bd10-2efb425f6718" />

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-native-capture-protection",
-  "version": "2.3.3",
+  "version": "2.3.4",
   "description": "🛡️ A React Native library to prevent and detect for screen capture, screenshots and app switcher for enhanced security. Fully compatible with both Expo and CLI.",
   "main": "./lib/commonjs/index.js",
   "module": "./lib/module/index.js",
@@ -26,7 +26,8 @@
     },
     "./package.json": "./package.json",
     "./app.plugin.js": "./app.plugin.js",
-    "./plugins/withPlugin": "./plugins/withPlugin.js"
+    "./plugins/withPlugin": "./plugins/withPlugin.js",
+    "./jest/capture-protection-mock": "./jest/capture-protection-mock.js"
   },
   "files": [
     "app.plugin.js",

